client/lib/application/src/folded-scope-list.jsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import _ from 'lodash';
+import Icon from 'react-fa';
+import 'common/asset/less/application/scope-list.less';
+
+class OwnerScopeList extends React.Component {
+    constructor(props) {
+        super();
```

Comment:
if constructor gets props you should pass them to super(props) as well

Reply:
Resolved only in this place. 
This is good point and this is very strange that linter didn't detect this issue. i've checked this and found similar issues in a lot of components. But i'm not sure that they all should be fixed because this can bring new unexpected issues. 
Please suggest if this fix is enough or you have some ideas regarding this?